pkgs/top-level/all-packages.nix

```diff
@@ -9662,8 +9662,6 @@ with pkgs;
 
   kzipmix = pkgsi686Linux.callPackage ../tools/compression/kzipmix { };
 
-  ma1sd = callPackage ../servers/ma1sd { };
```

Comment:
Add a throw for the attribute in pkgs/top-level/aliases.nix

Reply:
Done